catalog/transformer/catalog-transformer-pptx/src/main/java/ddf/catalog/transformer/input/pptx/PptxInputTransformer.java

```diff
@@ -117,6 +123,11 @@ private Metacard transformLogic(InputStream input, String id)
                     fileBackedOutputStream.asByteSource()
                             .openStream());
 
+            if(!titleAsMetadataTitle) {
+                // Allow the metacard title to be set as the filename
+                metacard.setAttribute(new AttributeImpl(Core.TITLE, (Serializable)null));
```

Comment:
Tika doesn't store the filename in the metadata, so at this point, it's unlikely we can set the filename directly